src/auth/manager/operation.go

```diff
@@ -0,0 +1,20 @@
+package manager
+
+type OpAction string
+type OpResource string
+
+var ActionRead OpAction = "read"
+var ActionWrite OpAction = "write"
+var ActionSign OpAction = "sign"
+var ActionDelete OpAction = "delete"
+var ActionDestroy OpAction = "destroy"
+
+var ResourceKey OpResource = "key"
```

Comment:
I think the resources should be plural. Also isn't `stores` missing?

Reply:
You r right, stores are missing. I will also use plurals and correlate that change with the permissions so the match

This vars were still not used since that will be added onces we implement the stores/nodes authorization.